roles/openldap/defaults/main.yml

```diff
@@ -0,0 +1,52 @@
+keystone:
```

Comment:
The namespace for the variables should match the role name, so in this case it should be openldap.  The variables here appear to be a mix of those specific for openldap as well as for keystone (ie. keystone_user_name).  The variables here should only be for openldap.  If we need to add users, we should enable that in a generic way.